addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -170,11 +167,19 @@ export class HtmlField extends Component {
         });
     }
 
+    computeContainsComplexHTML() {
+        // readonly if nodes in <head> as the head will be removed on insertion in the DOM
+        // this prevents the field from overwriting the value with invalid HTML when relevant
```

Comment:
Now that we're moving those comments around, I feel like it's the right time to wrap them at 80 chars, and add a capital letter at the start and a dot at the end. 😄